src/lecture/fourteen/frequencyFilter/Main.java

```diff
@@ -0,0 +1,27 @@
+package lecture.fourteen.frequencyFilter;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+        new BufferedReader(new InputStreamReader(System.in))
+                .lines()
+                .flatMap(x -> Stream.of(x.split("[^a-zA-Zа-яА-Я0-9]")))
```

Comment:
we can add:
```suggestion
                .flatMap(x -> Stream.of(x.split("[^a-zA-Zа-яА-Я0-9]+")))
```
and remove line 15